### ISSUE TITLE
sql: can't drop CHECK constraint

### ISSUE THREAD

**Issue by Gerold103** [tarantool/tarantool#4362]
```YAML
tarantool> box.execute('CREATE TABLE t1(id INTEGER PRIMARY KEY, CONSTRAINT ck1 CHECK(id > 0))')
---
- row_count: 1
...

tarantool> box.execute('ALTER TABLE t1 DROP CONSTRAINT ck1')
---
- error: Constraint CK1 does not exist
...

tarantool> box.space.T1.ck_constraint.CK1
---
- space_id: 512
  name: CK1
  expr: id > 0
...

```

**Comment by Korablev77** [comment]
I'd say it is duplicate of https://github.com/tarantool/tarantool/issues/4120 and blocked by https://github.com/tarantool/tarantool/issues/3503
In a nutshell, if we want to support ANSI syntax (which doesn't specify type of constraint being dropped) we should share one namespace for all constraints. Now we can create CHECK, FOREIGN KEY and UNIQUE constraints with the same name.

**Comment by kyukhin** [comment]
Closing as duplicate then.
